dora/core/common/src/main/java/alluxio/metrics/MultiDimensionalMetricsSystem.java

```diff
@@ -142,6 +145,15 @@ public static void initMetrics() {
     }
   }
 
+  /**
+   * Set the supplier for CACHE_STORAGE metrics.
+   *
+   * @param supplier supplier for cache storage
+   */
+  public static void setCacheStorageSupplier(DoubleSupplier supplier) {
+    sCacheStorageSupplier = supplier;
+  }
```

Comment:
allow clearing this static reference when the page meta store is closed. Otherwise, it cannot be GCed because the metrics system is holding a strong reference to it. Typically the meta store can hold up to several million `PageInfo` objects which consumes a considerable amount of memory.

Reply:
sounds good. where do you suggest adding the clearing?